docs/book/src/getting-started-with-aks.md

```diff
@@ -0,0 +1,120 @@
+# Getting started with cluster-api-provider-azure
+
+## Prerequisites
+
+### Requirements
+
+<!-- markdown-link-check-disable-next-line -->
+- A [Microsoft Azure account](https://azure.microsoft.com/)
+  - Note: If using a new subscription, make sure to [register](https://learn.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types) the following resource providers:
+    - `Microsoft.Compute`
+    - `Microsoft.Network`
+    - `Microsoft.ContainerService`
+    - `Microsoft.ManagedIdentity`
+    - `Microsoft.Authorization`
+    - `Microsoft.ResourceHealth` (if the `EXP_AKS_RESOURCE_HEALTH` feature flag is enabled)
+- Install the [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+- A [supported version](https://github.com/kubernetes-sigs/cluster-api-provider-azure#compatibility) of `clusterctl`
+
+### Setting up your Azure environment
+
+  1. Login with the Azure CLI.
+
+   ```bash
+  az login
+   ```
+
+  2. List your Azure subscriptions.
+
+   ```bash
+  az account list -o table
+   ```
+
+  3. If more than one account is present, select the account that you want to use.
+
+   ```bash
+  az account set -s <SubscriptionId>
+   ```
+
+  4. Save your Subscription ID in an environment variable.
+
+  ```bash
+  export AZURE_SUBSCRIPTION_ID="<SubscriptionId>"
+  ```
+## Creating an AKS Management Cluster with Workload Identity
+
+  1. Create an AKS Cluster with Workload Identity and OIDC Endpoint Enabled.
+  ```bash
+  az aks create \
+  --resource-group <resource-group-name> \
+  --name <aks-cluster-name> \
+  --enable-oidc-issuer \
+  --enable-workload-identity \
+  --node-count 2 \
+  --node-vm-size Standard_B2s \
+  --generate-ssh-keys \
+  --location <region>
+  ```
+  
+  2. Retrieve Credentials for the AKS Cluster to interact with it using kubectl:
+  ```bash
+  az aks get-credentials --resource-group <resource-group-name> --name <aks-cluster-name>
+  ```
+
+  3. Retrieve the OIDC Issuer URL and OIDC issuer URL.
+  ```bash
+  az aks show \
+  --resource-group <resource-group-name> \
+  --name <aks-cluster-name> \
+  --query "oidcIssuerProfile.issuerUrl" -o tsv
+  ```
+  Hold onto the OIDC issuer URL for creating federated credentials.
+
+  4. Create a User Assigned Managed Identity (UAMI) to use for Workload Identity.
```

Comment:
```suggestion
  4. Create a User-Assigned Managed Identity (UAMI) to use for Workload Identity.
```

"User-assigned" with a hyphen is correct English, although Azure docs don't always get it right. (This is my personal crusade; feel free to ignore.)